mkt/websites/indexers.py

```diff
@@ -0,0 +1,89 @@
+from operator import attrgetter
+
+from mkt.search.indexers import BaseIndexer
+from mkt.translations.models import attach_trans_dict
+from mkt.translations.utils import format_translation_es
+
+
+class WebsiteIndexer(BaseIndexer):
+    translated_fields = ('description', 'short_title', 'title', 'url')
+
+    @classmethod
+    def get_mapping_type_name(cls):
+        return 'mkt_website'
+
+    @classmethod
+    def get_model(cls):
+        """Returns the Django model this MappingType relates to"""
+        from mkt.websites.models import Website
+        return Website
+
+    @classmethod
+    def get_mapping(cls):
+        """Returns an Elasticsearch mapping for this MappingType"""
+        doc_type = cls.get_mapping_type_name()
+
+        mapping = {
+            doc_type: {
+                'properties': {
+                    'id': {'type': 'long'},
+                    'category': cls.string_not_analyzed(),
+                    'created': {'type': 'date', 'format': 'dateOptionalTime'},
+                    'description': {'type': 'string',
+                                    'analyzer': 'default_icu'},
+                    'default_locale': cls.string_not_indexed(),
+                    'icon_hash': cls.string_not_analyzed(),
+                    'icon_type': cls.string_not_analyzed(),
```

Comment:
Are we going to do filters or queries on these? If not, we can use the `no_index` variant so we don't fill up the index for something we're not using.